api/src/certification/session-management/domain/read-models/JuryCertificationSummary.js

```diff
@@ -54,8 +75,19 @@ function _getStatus({ status, isEndedBySupervisor }) {
 
   return status;
 }
+
+function _getCertification({ complementaryCertificationLabelTaken, complementaryCertificationKeyTaken }) {
```

Comment:
```suggestion
function _getCertificationType({ complementaryCertificationLabelTaken, complementaryCertificationKeyTaken }) {
```

ou `_getCertificationKey` ? 
Je sais pas trop, mais là avec `_getCertification`, j'ai l'impression qu'on reçoit l'objet complet :S